src/main/java/nextstep/subway/line/domain/LineColor.java

```diff
@@ -0,0 +1,52 @@
+package nextstep.subway.line.domain;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class LineColor {
+    private static final String NO_LINE_COLOR_ERROR = "지하철 노선 색이 필요 합니다.";
+
+    @Column
+    private String color;
```

Comment:
노선에 색상은 무조건 있어야하니 nullable = false 추가 해주시면 좋을 것 같습니다
```suggestion
    @Column(nullable = false)
    private String color;
```

Reply:
오 이건 제가 놓친 부분이네요 다음 단계에서 수정 하겠습니다!